dev-db/redis/redis-2.2.11.recipe

```diff
@@ -1,15 +1,32 @@
-HOMEPAGE="http://redis.io/"
+# WARNING: SUMMARY must be added to recipe here
 DESCRIPTION="
 Redis is an open source, advanced key-value store. It is often referred to as \
 a data structure server since keys can contain strings, hashes, lists, sets \
-and sorted sets.
-"
-SOURCE_URI="http://redis.googlecode.com/files/redis-2.2.11.tar.gz"
+and sorted sets."
+HOMEPAGE="http://redis.io/"
+COPYRIGHT="2006-2010 Salvatore Sanfilippo"
+LICENSE="BSD (3-clause)"
 REVISION="1"
-DEPEND=""
+SOURCE_URI="http://redis.googlecode.com/files/redis-2.2.11.tar.gz"
 CHECKSUM_MD5="5501c90ca02ae7ea93c34f067b3e0619"
```

Comment:
Not easily: it would require downloading the source file, checking that the MD5 checksum still matches, then computing the SHA256 checksum from the downloaded data.
Maybe it would be better to have the tool integrated in haikuporter or distributed separately ("haikuporter --modernize recipe" or similar) instead of trying to batch-fix everything at once. This way, it would be possible to make such more complex fixes.